piety-promoted-v2/modernized/17-chapter-16.adoc

```diff
@@ -243,7 +243,7 @@ peaceable resignation to the divine will.
 Among other expressions of weight which she uttered, the following are remembered:
 that she believed herself near her end,
 for she did not see that she had any thing more of religious duties to do,
-meaning of a public nature;`'For, '`said she, when I look at our meetings,
+meaning of a public nature; `'For,`' said she, when I look at our meetings,
```

Comment:
The quotations at the end of this paragraph need a bit more work, we need one before `when I` as well, and the spacing around `added` on line 249 below is off.  Here's a screenshot from the original, for the sake of matching it:

![2020-02-25_11-49-49](https://user-images.githubusercontent.com/7050938/75268120-1c1c1e00-57c5-11ea-8c9e-9db2653e446b.png)